src/nvim/ex_cmds.c

```diff
@@ -373,6 +374,8 @@ void ex_sort(exarg_T *eap)
       sort_rx = TRUE;
     else if (*p == 'n')
       sort_nr = 2;
+    else if (*p == 'b')
+      sort_bin = 2;
```

Comment:
Braces, here and in all of the other lines of this if. Though I would really convert this to `switch`